hybrid/compartment_examples/inter_comp_call/malicious_compartments/secure-redirect_clr/shared.S

```diff
@@ -11,4 +11,5 @@ executive_switch:
     mov      x0, #0
     cvtp     clr, lr
```

Comment:
@0152la As far as I can tell, this instruction can be removed too at this point (I don't think we need `lr` converted to a capability).

Reply:
I *think* this is needed to revert the capability metadata set in `switch_compartment` itself [1]. However, without a `gdb` in front of me going instruction by instruction, I can't say certainly that this cannot be remove. The program *should* crash after returning from `switch_compartment` [2] if we do not explicitly provide a capability. Also should add if this `cvtp` is removed, the corresponding return in `switch_compartment` [2] needs to be changed to a simple `ret`.

[1] https://github.com/gabi-250/cheri-examples/blob/bd4d47f71537402261bd4b4bd007322ab0c0e71a/hybrid/compartment_examples/inter_comp_call/malicious_compartments/shared/switch_compartment.S#L66
[2] https://github.com/gabi-250/cheri-examples/blob/bd4d47f71537402261bd4b4bd007322ab0c0e71a/hybrid/compartment_examples/inter_comp_call/malicious_compartments/shared/switch_compartment.S#L79

Reply:
Alright, so can I read that as we both agree the `cvtp` is needed then?

Reply:
yep!